styles/gradient.css

```diff
@@ -37,3 +37,7 @@
 .text-gradient-pooltogether-link {
   @apply text-purple-800 dark:text-purple-500;
 }
+
+.bg-gradient-connext {
+  @apply bg-gradient-to-r from-purple-800 to-indigo-800 dark:from-purple-600 dark:to-indigo-500;
+}
```

Comment:
we're removing syles related to a specific integration to the shared folders, so you should either move it inline or add it to the connext integration folder